api/api_templates.gen.go

```diff
@@ -193,7 +193,7 @@ func (a *TemplatesApiService) ApplyTemplateExecuteWithHttpInfo(r ApiApplyTemplat
 		newErr.body = localVarBody
 		newErr.error = localVarHTTPResponse.Status
 		if localVarHTTPResponse.StatusCode == 422 {
-			var v TemplateSummary
+			var v TemplateSummaryError
```

Comment:
Does TemplateSummaryError implement SetMessage 'properly'?

Reply:
(i.e. if we get an error back from `a.client.decode`, will that be reflected properly in the output?

Reply:
Yes, it implements it via code generation now